### DISCUSSION TITLE
Question; should 10.5281/zenodo.7035342 work as an import, now or later or never?

### DISCUSSION THREAD

**Discussion by mietcls** [ugent-library/biblio-backend#1366] [Announcements]
Steps to reproduce:

Add publication via DOI
![Image](https://user-images.githubusercontent.com/1572885/198229385-bcbb135d-7342-4ed2-8392-8d8d5b8e1a2b.png)

Get error
![Image](https://user-images.githubusercontent.com/1572885/198229483-a48b8fef-7cc6-415f-afa6-924673fbcf07.png)

**Comment by nics** [comment]
error:
```
import single publication: could not fetch publication	{"errors": "can't import publication: Resource not found.", "publication": "10.5281/zenodo.7035342", "user": "F72763F2-F0ED-11E1-A9DE-61C894A0A6B4"}
```

**Comment by nics** [comment]
@mietcls not findable through crossref, will have to be fixed later

**Comment by mietcls** [comment]
@nics Stefanie just provided this information, since this DOI is provided via Zenodo / DataCite.

CrossRef is a central organisation that handles publications only. When a publisher creates a DOI for a publication, he will do that through CrossRef (using the CrossRef metadata schema).

The Zenodo repository allows to deposit different outputs, not only data but also publications. However, Zenodo always uses DataCite as a central organisation for the creation of DOIs, also for publications. The metadata schema of DataCite is different from that of CrossRef. The resourceType in the DataCite metadata denotes whether it is a publication.

Currently, we have decided not to put a filter on resourceType in the Biblio dataset workflow, but the guidelines will state that publications are not in scope and the curators will take that check into account during curation. Publications on publisher's websites have most of the time undergone a peer review process while publications in Zenodo have not. The peer-reviewed journal articles are the focus of the publication policy and the reporting for the Open Science KPI (FRIS). We could in the future consider expanding the functionality of the Biblio publication workflow to cover the DOI-based import of these non-peer-reviewed publications from Zenodo for instance, but that would need discussion and investigation on whether it is worth it.

For now, they can be added manually if needed.

Edited in October, remarked again Wednesday 8 March 2023.

**Comment by nicolasfranck** [comment]
See https://github.com/ugent-library/biblio-backoffice/issues/1138
See https://github.com/ugent-library/biblio-backoffice/pull/1163

**Comment by mietcls** [comment]
Related https://github.com/ugent-library/biblio-backoffice/discussions/1185
